test/e2e/tsconfig.json

```diff
@@ -6,5 +6,10 @@
     "lib": ["es6", "dom"],
     "types": ["node"],
   },
-  "include": ["**/*.ts"]
+  "include": ["**/*.ts"],
+  // "references": [
```

Comment:
I had issues with this bit of code on my local and had to comment this out. Can someone verify this?